imgui_test_suite/imgui_tests_widgets_inputtext.cpp

```diff
@@ -679,33 +679,40 @@ void RegisterTests_WidgetsInputText(ImGuiTestEngine* e)
                 KeyPressAndDebugPrint(chord_word_prev);
                 IM_CHECK_EQ(state->GetCursorPos(), 6 + 5 + 2 + 3);          //  "Hello world. Foo|.bar!!!" // VS-Win: STOP, GitHubWeb-Win: SKIP
 
-                KeyPressAndDebugPrint(chord_word_prev;
+                KeyPressAndDebugPrint(chord_word_prev);
 #endif
                 IM_CHECK_EQ(state->GetCursorPos(), 6 + 5 + 2);              //  "Hello world. |Foo.bar!!!"
 
                 KeyPressAndDebugPrint(chord_word_prev);
-#if IMGUI_BROKEN_TESTS // may be fixed by #6067
+#if IMGUI_VERSION_NUM >= 18930
                 IM_CHECK_EQ(state->GetCursorPos(), 6 + 5);                  //  "Hello world|. Foo.bar!!!" // VS-Win: STOP, GitHubWeb-Win: SKIP
                 KeyPressAndDebugPrint(chord_word_prev);
 #endif
                 IM_CHECK_EQ(state->GetCursorPos(), 6);                      //  "Hello |world. Foo.bar!!!"
 
                 KeyPressAndDebugPrint(chord_word_prev);
                 IM_CHECK_EQ(state->GetCursorPos(), 0);                      // "|Hello world. Foo.bar!!!"
+
+                KeyPressAndDebugPrint(chord_word_prev);
+                IM_CHECK_EQ(state->GetCursorPos(), 0);                      // "|Hello world. Foo.bar!!!" // (no-op)
             }
             else
             {
                 // OSX
                 KeyPressAndDebugPrint(chord_word_next);
                 IM_CHECK_EQ(state->GetCursorPos(), 5); //  "Hello|"
+                ImGui::SetActiveID(state->ID, ImGui::GetCurrentWindow());
```

Comment:
These lines were necessary for me to test on Windows since the Alt keypress will change the focus to the dropdown arrow, causing the next keypress to not be registered by the inputtext widget. Of course a more robust solution is needed but I couldn't find one quickly.

Reply:
That part was actually a bug on OSX which was fixed by [fc50532](https://github.com/ocornut/imgui/commit/fc50532) around the time you made that PR, so I'm removing those lines now.